libsplinter/src/biome/client/mod.rs

```diff
@@ -64,7 +64,7 @@ pub struct UpdateUser {
 }
 
 /// New key pairs to be added while updating a Biome user.
-#[derive(Debug)]
+#[derive(Debug, PartialEq, Eq, Clone)]
```

Comment:
We should start getting in the habit of doing this as `#[cfg_attr(test, derive(/*traits for tests only*/))]`, that way we're not including these things in the public API if it isn't necessary.

Reply:
> We should start getting in the habit of doing this as `#[cfg_attr(test, derive(/*traits for tests only*/))]`, that way we're not including these things in the public API if it isn't necessary.

With the possible exception of Debug, wouldn't we want to expose these trait implementations?